### ISSUE TITLE
[BUG]: Onimusha 3, ako/samanosuke teleport vfx and pink barrier mist vfx messed up

### ISSUE THREAD

**Issue by washingleavesandthensome23-434** [PCSX2/pcsx2#8983]
### Describe the Bug

Akos/Samanosukes teleport vfx is messed up  also pink barrier mist vfx are messed

### Reproduction Steps

Play Paris level from start 

### Expected Behavior

Not having said vfx issues

### PCSX2 Revision

1.7.4583

### Operating System

Windows 11

### CPU

I9-13900K

### GPU

4090

### GS Window Screenshots

![150457cb-0cfc-4dd2-91b6-3d9a60d60b50](https://github.com/PCSX2/pcsx2/assets/60486562/fecbaaf9-6096-42f7-b9be-964cccb44c7c)

![0b4fefe5-8cfc-41a8-87aa-6313d7c12a1a](https://github.com/PCSX2/pcsx2/assets/60486562/cd59f31b-89a7-47a0-8290-475febeb1d7c)

![d28823b4-91d1-4252-bbf7-efcc1704d700](https://github.com/PCSX2/pcsx2/assets/60486562/784b19e4-1e45-4f8a-a6fd-aaa2055091cc)

**Comment by Mrlinkwii** [comment]
can i have a gs dump

**Comment by washingleavesandthensome23-434** [comment]
A sec

**Comment by washingleavesandthensome23-434** [comment]
Here

**Comment by stenzek** [comment]
That GS dump looks pretty much identical in HW and SW, and also doesn't match the screenshot above.

**Comment by washingleavesandthensome23-434** [comment]
K, lemme get a better gsdump

**Comment by washingleavesandthensome23-434** [comment]
K and fresh dumps also updated screen shots

**Comment by Mrlinkwii** [comment]
this gs dump is the same in HW/SW ( as i can see to say , dont upscale) 
![image](https://github.com/PCSX2/pcsx2/assets/17685135/4cef6699-9480-48c2-981d-9a87954f03fe)

while this gs dump dose have an issue in HW  
![image](https://github.com/PCSX2/pcsx2/assets/17685135/34f8cf36-7397-4b53-98cc-41b491bfbb29)

**Comment by washingleavesandthensome23-434** [comment]
Bout the red mist perhaps I wasn't clear enough,  a sec

the issue is clearly present:

**Comment by Mrlinkwii** [comment]
@washingleavesandthensome23-434  can you reupload that gs dump again , it seemed to fail uploading

**Comment by washingleavesandthensome23-434** [comment]
Updated

**Comment by washingleavesandthensome23-434** [comment]
@devs, Hope ya got everything needed too fix these bugs 🙂

**Comment by stenzek** [comment]
A bisect to the exact build which broke it would be helpful.

My usual go-to is to compare hardware to software to pinpoint what the issue is, but if software's broken, but was previously fine, we'll need to know when that changed.

**Comment by DrunkRichtofen** [comment]
I'm currently running through earlier builds to narrow down which build introduced this bug. So far, I've found that both vfx bugs were not present in build 1.7.4308, so this occured within the last 2 months at most. I'll run through more builds and try to narrow it down.

**Comment by washingleavesandthensome23-434** [comment]
Appreciated man this helps out alot

**Comment by DrunkRichtofen** [comment]
I've managed to find the exact builds that these regressions were introduced in. The red fog bug appeared in v1.7.4428, while the Time Fold and Ako teleport bugs both appeared in v1.7.4438

**Comment by washingleavesandthensome23-434** [comment]
Holy f! Very nice my dude!

**Comment by washingleavesandthensome23-434** [comment]
Btw for those interested in what the overlay on the barrier is, The weird square gradient that showed in recent builds, is actually from the item preview textures
They are the only textures in the entire game that has that watermark

**Comment by stenzek** [comment]
Can you try ingame with https://github.com/PCSX2/pcsx2/pull/9076 ?

Since the texture data has already been corrupted when you made those GS dumps, it's still busted when playing them back, but hopefully it'll be correct if you fresh boot the game (don't use a save state).

**Comment by washingleavesandthensome23-434** [comment]
Well in order too test I'll need an updated build right? I'll have too wait on merge into master

**Comment by refractionpcsx2** [comment]
You can get a build from the checks at the bottom, just press details next to one of the windows builds with a tick, then go to summary

**Comment by washingleavesandthensome23-434** [comment]
Ngl I'm confused asf , im just moving in circles cant find download 😥

**Comment by refractionpcsx2** [comment]
*sigh*

Fine, just get this https://github.com/PCSX2/pcsx2/suites/13845279791/artifacts/768979923

**Comment by washingleavesandthensome23-434** [comment]
Lol, appreciated 🫡

**Comment by refractionpcsx2** [comment]
not really concerned about the texture dumping, does the game look right?

**Comment by washingleavesandthensome23-434** [comment]
Yup all 3 issues fixed🫡😎

**Comment by washingleavesandthensome23-434** [comment]
Ya done good @devs, appreciated very much for fixing 🫡
